pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -226,7 +226,9 @@ func NewCmdRemoveClusterRoleFromUser(name, fullName string, f *clientcmd.Factory
 }
 
 func (o *RoleModificationOptions) CompleteUserWithSA(f *clientcmd.Factory, args []string, saNames []string) error {
-	if (len(args) < 2) && (len(saNames) == 0) {
+	if (len(args) < 1) && (len(saNames) == 1) {
```

Comment:
and actually, put the `if (len(args) < 2) && (len(saNames) == 0)` check after we've appended args to `o.Users` and check `if (len(o.Users) == 0) && (len(saNames) == 0)` instead